test/SparqlAntlrParserTestHelpers.h

```diff
@@ -1190,4 +1191,143 @@ inline auto ExistsFilter =
 
 }  // namespace matchers
 
+namespace {
```

Comment:
The anonymous namespace in a header doesn't make too much sense.
put those into an explicit namespace `sparqlParserTestHelpers` and then do a 
`using namespace sparqlParserTestHelpers` in the involved `.cpp` files.